src/WebApi/Controllers/ItemsController.cs

```diff
@@ -19,6 +21,22 @@ public class ItemsController : BaseController
     public Task<ActionResult<Result<IList<ItemViewModel>>>> GetItemsList() =>
         ResultToActionAsync(Mediator.Send(new GetItemsQuery()));
 
+    /// <summary>
+    /// Get items sharing the same BaseId.
+    /// </summary>
+    /// <param name="baseId">Item BaseId.</param>
+    /// <returns>The items sharing the same BaseId.</returns>
+    /// <response code="200">Ok.</response>
+    /// response code="400">Bad Request.</response>
+    [HttpGet("upgrades/{baseid}")]
```

Comment:
The route parameter need to match exactly the name the argument name (i.e. `baseId`)

Reply:
done